kanban-board/src/components/Issue/Issue.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Card,
+  CardContent,
+  Typography,
+  CardHeader,
+  Avatar,
+} from "@material-ui/core";
+import { Component, ReactNode } from "react";
+import { dragBegin } from "../../common/actions";
+import { Status } from "../../common/constants";
+import { IIssue } from "../../common/types";
+import { AppDispatch } from "../../store/store";
+import './Issue.css';
+
+interface IIssueProps {
+  issue: IIssue,
+  id: number,
+  dispatch: AppDispatch,
```

Comment:
This is a very interesting way of connecting the component to the store to me. In Datadash we use **mapDispatchToProp** in **connect()**.

I would have done it one of two ways:
1. connect Issue component to the store
2. dispatch an action in the parent and pass a parent methods